src/mantle/testing/wordpress-bootstrap.php

```diff
@@ -31,13 +33,56 @@
 // Load the configuration.
 if ( defined( 'WP_TESTS_CONFIG_FILE_PATH' ) && ! empty( WP_TESTS_CONFIG_FILE_PATH ) && is_readable( WP_TESTS_CONFIG_FILE_PATH ) ) {
 	$config_file_path = WP_TESTS_CONFIG_FILE_PATH;
+} elseif ( false === strpos( __DIR__, '/wp-content/' ) ) {
+	/**
+	 * Install WordPress automatically in the /tmp/wordpress folder.
+	 *
+	 * Retrieves the latest installation command from Mantle's GitHub and runs it
+	 * to install WordPress to a temporary folder (/tmp/wordpress by default).
+	 * This unlocks the ability to run `composer run phpunit` both locally and in
+	 * CI tests.
+	 */
+
+	defined( 'WP_TESTS_INSTALL_PATH' ) || define( 'WP_TESTS_INSTALL_PATH', '/tmp/wordpress' );
+
+	$config_file_path = WP_TESTS_INSTALL_PATH . '/wp-tests-config.php';
+
+	// Install WordPress if we're not in the sub-process that installs WordPress.
+	if ( ! defined( 'WP_INSTALLING' ) || ! WP_INSTALLING ) {
+		echo 'WordPress installation not found, installing in temporary directory: ' . WP_TESTS_INSTALL_PATH . PHP_EOL;
+
+		$dir = get_framework_directory();
+
+		// Download the latest installation command from GitHub and install WordPress.
+		$cmd = sprintf(
+			'WP_CORE_DIR=%s curl -s %s | bash -s %s %s %s %s %s %s',
+			WP_TESTS_INSTALL_PATH,
+			'https://raw.githubusercontent.com/alleyinteractive/mantle-framework/main/bin/install-wp-tests.sh',
```

Comment:
Because of `main`, this isn't a permalink. If we at some point move the file, or remove it, it would immediately break every mantle installation's integration of this code. On the other hand, having a dynamic reference carries benefits too, e.g. we can improve it or fix a bug and every codebase using it instantly benefits. Overall I think the cons of a dynamic reference outweigh the pros, so I'd recommend going that route. Two thoughts/ideas:

1. Instead of `main`, set it to a tag. That would be more future-proof -- but not entirely so. e.g. what if we renamed the repo?
2. Consider removing the shell script from this repo entirely and moving it to another one. It could even be, and likely should be, its own repo. Maybe there would even be value in pulling this file (or some of it) and the shell script into a separate composer package? Regardless, if that shell script had its own home, it would be safer to reference without thinking about it moving.

Reply:
One idea would be to at least use the `HEAD` version instead of `main`: https://raw.githubusercontent.com/alleyinteractive/mantle-framework/HEAD/bin/install-wp-tests.sh Homebrew uses a standalone repository https://github.com/Homebrew/install .

#2 seems like a better option to decouple the two things

Reply:
Ended up going with https://raw.githubusercontent.com/alleyinteractive/mantle-ci/HEAD/install-wp-tests.sh !